composition.js

```diff
@@ -0,0 +1,196 @@
+"use strict"
+
+const fs = require('fs')
+
+class Cookie {
+    constructor (ingredients) {
+        this._name = 'cookie'
+        this._status = 'mentah'
+        this._ingredients = []
+    }
```

Comment:
```suggestion
       this.setIngredient (ingredients)
    }
```

setIngredient langsung dipanggil saja di constructor.